Tools/machines/pitzer-osc/pitzer_warpx.profile.example

```diff
@@ -0,0 +1,68 @@
+# please set your project account
+export proj=""  # change me!
+
+# remembers the location of this script
+export MY_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+if [ -z ${proj-} ]; then echo "WARNING: The 'proj' variable is not yet set in your $MY_PROFILE file! Please edit its line 2 to continue!"; return; fi
+
+module purge
+# load these modules for optimization (todo: make this compile)
+# module load cmake/3.25.2
+# module load intel/19.0.5
+# module load cuda/11.6.1
+# module load mvapich2-gdr/2.3.7
+# export CC=$(which icc)
+# export CXX=$(which icpc)
+# export FC=$(which ifort)
```

Comment:
The CUDA version here is older and ICC/ICPC is end-of-life. Do you want to remove this comment block?
```suggestion
```

Reply:
I was about to use CUDA-aware MPI for the GPU build, but the system-installed packages on Pitzer are either outdated or have compatibility issues. The cluster was built in 2020, and the modules are from around that time frame. Therefore, I switched to self-installed OpenMPI 5.0.5 and HDF5 1.14.4 in my recent commits.

Reply:
Ah, that's too bad! Good work-around. Are the HPC admins responsive about this issue?

Reply:
Since you mentioned it, I tried again and found a compatible solution using the built-in CUDA-aware MPI. But I have to switch to the Intel compiler instead of GNU. This change is reflected in the latest commit.